src/pudl/output/ferc1.py

```diff
@@ -2274,36 +2274,45 @@ def annotated_forest(self: Self) -> nx.DiGraph:
         annotated_forest = deepcopy(self.forest)
         nx.set_node_attributes(annotated_forest, self.node_attrs)
         nx.set_edge_attributes(annotated_forest, self.edge_attrs)
-        annotated_forest = self.propagate_node_attributes(annotated_forest)
 
         logger.info("Checking whether any pruned nodes were also tagged.")
         self.check_lost_tags(lost_nodes=self.pruned)
         logger.info("Checking whether any orphaned nodes were also tagged.")
         self.check_lost_tags(lost_nodes=self.orphans)
-        self.check_conflicting_tags(annotated_forest)
+
+        annotated_forest = self.propagate_node_attributes(annotated_forest)
         return annotated_forest
 
     def propagate_node_attributes(self: Self, annotated_forest: nx.DiGraph):
         """Propagate tags.
 
-        Propagate tags leafwards, rootward &  to the _correction nodes.
+        Propagate tags root-ward, leaf-wards &  to the _correction nodes. We
+        propagate the tags root-ward first because we primarily manually
+        compiled tags for the leaf nodes, so we want to send those leafy tags
+        root-ward first before trying to send tags back up the graph.
         """
-        ## Leafwards propagation
-        annotated_forest = _propagate_tags_leafward(annotated_forest, ["in_rate_base"])
-        # Rootward propagation
-        annotated_forest = _propagate_tag_rootward(annotated_forest, "in_rate_base")
+        tags_to_propagate = ["in_rate_base", "rate_base_category"]
+        # Root-ward propagation
+        for tag in tags_to_propagate:
+            annotated_forest = _propagate_tag_rootward(annotated_forest, tag)
+        ## Leaf-wards propagation
+        annotated_forest = _propagate_tags_leafward(annotated_forest, tags_to_propagate)
```

Comment:
this change does two things:
* adds another tag that we are now propagating
* sends the tag from the leaves towards the roots first (see the docs above for why)

Reply:
yes it is. the `_propagate_tag_rootward` tags one `tag` while the `_propagate_tags_leafward` tags a list of tags